cryptoportfoliotracker.py

```diff
@@ -1,81 +1,78 @@
+import cli
+
+from bs4 import BeautifulSoup  # type: ignore
 from os import getcwd, path, system
-from selenium import webdriver
-from selenium.webdriver.firefox.options import Options
-from bs4 import BeautifulSoup
+from selenium import webdriver  # type: ignore
+from selenium.webdriver.firefox.options import Options  # type: ignore
+from selenium.webdriver.firefox.webdriver import WebDriver  # type: ignore
+from typing import List
 
-import config
-import cli
 from coin import Coin
+from config import CFG
 
 
-def get_coins():
+def get_coins() -> List[Coin]:
     """
     Creates coin objects for each coin entered in config.py and returns a list containing all coin objects
     :return: coins (list of coin objects)
     """
-    coins = []
-    for key in config.bsc_portfolio:
-        coin = Coin()
-        coin.set_coin_type("bsc")
-        coin.set_contract(key)
-        coin.set_quantity(config.bsc_portfolio.get(key))
-        coin.set_tracker_url()
-        coins.append(coin)
-    for key in config.eth_portfolio:
-        coin = Coin()
-        coin.set_coin_type("eth")
-        coin.set_contract(key)
-        coin.set_quantity(config.eth_portfolio.get(key))
-        coin.set_tracker_url()
-        coins.append(coin)
+    coins: List[Coin] = []
+
+    for portfolio in CFG.portfolios:
+        for con in portfolio.contracts:
```

Comment:
Specifically here I thought that a nested for is better than two near-identical for loops since the data was very same-y.